src/hssm/hssm.py

```diff
@@ -46,9 +47,9 @@
 
     import arviz as az
     import pandas as pd
-    import pytensor
 
 LogLikeFunc = Callable[..., ArrayLike]
+pytensor.config.floatX = "float32"
```

Comment:
I am slightly wary about changing a global option under the hood that affects more than HSSM, so I will defer to @AlexanderFengler for decision on this. We should at least add a warning that this option is changed just so that users know we changed this and to play well with other packages

Reply:
I am not sure if this is still relevant here (says outdated and during my review I didn't encounter this in the diff anymore. 

But in general, yeah we could at least a warning or print statement that let's users know that this setting was changed. 

We could instead have this code explicit in all of our examples, but that wouldn't look as clean, so slightly biased against it.

Reply:
@AlexanderFengler check out #161. Was thinking of adding a convenience function in `hssm` so float types can be changed explicitly (and also `pytensor` and `jax`) at the same time. Can be a good compromise